styles.css

```diff
@@ -118,3 +122,353 @@ body {
   top: 250px;
   transform: scale(0.85);
 }
+
+/* Middle Section */
+.works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 120px 0;
+  gap: 20px;
+  background: #e5e5e5;
+  position: absolute;
+  width: 375px;
+  height: 3455px;
+  left: 0;
+  top: 682px;
+}
```

Comment:
- You did great  our structure looks nice also your design but I just suggest a small change in the background color kindly use that one    ` background: #FFFFFF;`  instead of  this ` background: #e5e5e5;` in the works section so that it will make your design stick more to the Figma design as required in the project :+1: 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/205385422-d39d682c-d436-4911-ac5c-fd65dc744430.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/205385617-898a8dd9-bc9e-4a70-ba44-4d89d453d623.png)